bazel/repository_locations.bzl

```diff
@@ -459,6 +459,13 @@ GIT_REPOSITORY_LOCATIONS = dict(
         commit = "fa36a4fa141b4e9486b9126640d54a94c1d36fce",
         shallow_since = "1638898188 -0800",
     ),
+
+    # Used by the perf profiler to create pprof proto output.
+    com_github_google_pprof = dict(
```

Comment:
Please use http repos not git repos. git repos don't use the repository cache

Reply:
Hi @JamesMBartlett,

Could you take a look at the upstream pprof repo on github? I did not see any tags or any other artifacts that looked compatible with `http_archive` or `_bazel_repo`. Maybe there is a way to make it work, but it was not obvious to me. (or maybe you are suggesting a different way to bring the repo in). I had left a comment (above) where I invoked `_git_repo`:
```
# Using "git repo" because the pprof repo does not have any release tags.
```

Reply:
We do this in a number of places in this file, you can just point to the commit itself. It's not ideal because github changes their hashes sometimes, but that's a separate issue we need to figure out. 
eg: "https://github.com/antlr/grammars-v4/archive/e53d7a1228505bfc80d8637808ef60e7eea92cc2.tar.gz"